utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -78,9 +82,25 @@ class UrlImageParser private constructor(
     }
   }
 
-  private fun calculateInitialMargin(drawableWidth: Int): Int {
-    val availableAreaWidth = htmlContentTextView.width
-    return (availableAreaWidth - drawableWidth) / 2
+  private fun calculateInitialMargin(availableAreaWidth: Int, drawableWidth: Int): Int {
+    val margin = (availableAreaWidth - drawableWidth) / 2
+    return if (margin > 0) {
+      margin
+    } else {
+      0
+    }
+  }
+
+  // Reference: https://stackoverflow.com/a/51865494
+  fun <T : View> T.width(calculateWidth: (Int) -> Unit) {
+    if (width == 0)
```

Comment:
Nit: please add curly braces since this is a large block of code.

Reply:
Added curly braces for if/else block